src/sauce-testcafe-config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
```

Comment:
Better make lodash an explicit dependency in package.json

Reply:
👍